debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/structureformat/extensions/J9ObjectStructureFormatter.java

```diff
@@ -89,17 +90,24 @@ public FormatWalkResult format(String type, long address, PrintStream out,
 				if (className.equals("java/lang/String")) {
 					formatStringObject(out, 0, clazz, dataStart, object, address);
 				} else if (isArray) {
-					int begin = DEFAULT_ARRAY_FORMAT_BEGIN;
-					int end = DEFAULT_ARRAY_FORMAT_END;
-					if (extraArgs.length > 0) {
-						begin = Integer.parseInt(extraArgs[0]);
-					}
-
-					if (extraArgs.length > 1) {
-						end = Integer.parseInt(extraArgs[1]);
+					if (ValueTypeHelper.getValueTypeHelper().isJ9ClassIsFlattened(clazz) 
```

Comment:
Please remove trailing whitespace from this line and any others in the files you're modifying.